### PR TITLE
Fuse tas rebased

### PR DESCRIPTION
**Pull request by CarlOlsson**
Rebased #58 on master.
Fusion of airspeed still needs some small fixes.
E.g. when to fuse airspeed (now we never do it) and how often we want to do it.
@tumbili can you upload the plots from today's flight?

### CODE REVIEW

**Comment by mcsauder** [comment]
Hi Carl,

I think your PR will pass build checks if you change lines 92-93

    memset(_airspeed_innov, 0, sizeof(_airspeed_innov));
    memset(_airspeed_innov_var, 0, sizeof(_airspeed_innov_var));

to mirror lines 77-78, since these values are just floats and not arrays of floats.  Be sure to place them in the initializer list above the _heading_innov as you have done in the ekf.h file, to keep the initialization order and avoid compiler warnings.

    _airspeed_innov(0.0f),
    _airspeed_innov_var(0.0f),
    _hagl_innov(0.0f),
    _hagl_innov_var(0.0f),
        
I had to learn these lessons in the past few weeks by the school of hard knocks... I hope this helps!

**Comment by priseborough** [comment]
I think the next steps are to get the airspeed data added to the logging and replay function and get the control logic implemented so that airspeed fusion does not run at inappropriate times (eg on ground) and the wind states and associated covariances are initialised correctly when airspeed fusion is started.

**Comment by CarlOlsson** [comment]
@mcsauder @priseborough 
Thank you very much for the comments!
I changed the initialization to not use memset, hope it passes the checks now

**Comment by tumbili** [comment]
airspeed_innov

**Comment by tumbili** [comment]
wind

**Comment by tumbili** [comment]
Log: http://logs.uaventure.com/view/3Yvra5Fj889soKNEjCYWw

**Comment by tumbili** [comment]
@priseborough I would suggest that we have this merged but not used yet in the estimator. This avoids pailful rebasing. @CarlOlsson Can you implement the steps which @priseborough has proposed? I think Paul is always open to have a chat on skype if you get stuck somewhere.

**Comment by priseborough** [comment]
Happy with that proposal provided it is off by default until we get the logic and data logging in and tested.

**Comment by CarlOlsson** [comment]
Yes I am looking forward to implementing the steps. Will create a todo list for it.

File: EKF/estimator_interface.cpp [hunk 1]
```diff
@@ -203,11 +203,11 @@ void EstimatorInterface::setAirspeedData(uint64_t time_usec, float *data)
 		return;
 	}
 
-	if (time_usec > _time_last_airspeed) {
+	if (time_usec - _time_last_airspeed > 1000000) { //Limit the airspeed update rate to 1 Hz for now
```
Comment (tumbili) [inline EKF/estimator_interface.cpp]: @CarlOlsson We should increase this!

**Comment by tumbili** [comment]
@CarlOlsson Where is this standing? Let me know if I can help you somehow. It would be great if we can merge this even if airspeed fusion is not activated yet.

**Comment by CarlOlsson** [comment]
@tumbili I thought we decided on merging this? (To avoid future rebasing) I have a [todo list](#77) that needs to be done before we can activate tas fusion

**Pull request closed without merging**